spec/factories/content_item.rb

```diff
@@ -83,26 +83,22 @@
       end
 
       trait :by_auth_bypass_id do
-        access_limited {
-          {
-            "auth_bypass_ids" => %w(85aa9fd5-c514-4964-b931-5b597e4ec668),
-          }
-        }
+        auth_bypass_ids { %w(85aa9fd5-c514-4964-b931-5b597e4ec668) }
       end
 
       trait :by_auth_bypass_id_and_user_id do
+        auth_bypass_ids { %w(85aa9fd5-c514-4964-b931-5b597e4ec668) }
         access_limited {
           {
-            "auth_bypass_ids" => %w(85aa9fd5-c514-4964-b931-5b597e4ec668),
             "users" => %w(M6GdNZggrbGiJrLjMSbKqA f17250b0-7540-0131-f036-005056030202),
           }
         }
       end
 
       trait :by_auth_bypass_id_and_org_id do
+        auth_bypass_ids { %w(85aa9fd5-c514-4964-b931-5b597e4ec668) }
         access_limited {
           {
-            "auth_bypass_ids" => %w(85aa9fd5-c514-4964-b931-5b597e4ec668),
             "organisations" => %w(f17250b0-7540-0131-f036-005056030202),
           }
```

Comment:
I'd suggest moving these traits to be separate from this `access_limited_content_item` factory.

I think it'd be be consistent to have a `with_auth_bypass_id` trait on the content item that defaults to a Random UUID. (similar to https://github.com/alphagov/content-store/blob/59a14cd8103340492638c3ea6494416824c8a10b/spec/factories/content_item.rb#L15-L17)